it/datadog/src/main/java/com/google/cloud/teleport/it/datadog/DatadogLogEntry.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (C) 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.it.datadog;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.auto.value.AutoValue;
+import javax.annotation.Nullable;
+
+/** A class for Datadog log entries, copy of DatadogEvent. */
+@AutoValue
+public abstract class DatadogLogEntry {
```

Comment:
This should be replaced by `DatadogEvent` when we get around to contributing the `DatadogIO` to Apache Beam.